### DISCUSSION TITLE
How to correct the indentation after breaking the line in the middle with R + Enter in vim mode

### DISCUSSION THREAD

**Discussion by mynanshan** [zed-industries/zed#19442] [Feature Requests]
I am using Zed 0.157.5 on Ubuntu 22.04. Here is a minimal Python example of the problem I observed:
![image](https://github.com/user-attachments/assets/65eb79bf-853c-48c1-bf5f-2610dad4c821)
With my cursor placed where as shown in the screenshot, I hit `R` + `Enter` and it became this:
![image](https://github.com/user-attachments/assets/5a58b7a7-b03e-40b4-88e9-81caad783da8)
Note that the part after the line break position should be aligned with `(`, but it isn't.

In contrast, the indentation will be correct if I use `Enter` in the insert mode to break the line.

I am wondering if there are any ideas on how to fix the indentation behavior.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
